view-creator-framework/build.gradle.kts

```diff
@@ -82,3 +80,6 @@ tasks.named<org.hypertrace.gradle.avro.CheckAvroCompatibility>("avroCompatibilit
 }
 
 group = "org.hypertrace.core.viewcreator"
+repositories {
```

Comment:
we already have it in `https://github.com/hypertrace/view-generator-framework/blob/main/settings.gradle.kts`. Prior, it was not needed explicitly, and the same set of libs are updated. Can you remove this block and retry?